src/main/core-options.js

```diff
@@ -115,15 +115,15 @@ const options = {
     category: CATEGORY_GLOBAL,
     type: "choice",
     default: [
-      { since: "0.0.10", value: "babylon" },
+      { since: "0.0.10", value: "babel" },
```

Comment:
We do not need to update it here since it _was_ `babylon` from `0.0.10` to `1.13.0`.

```suggestion
      { since: "0.0.10", value: "babylon" },
```